apps/services/endorsements/api/src/environments/environment.ts

```diff
@@ -66,8 +66,11 @@ const prodConfig = {
     Admin: process.env.ACCESS_GROUP_ADMIN ?? '',
   },
   email: {
-    sender: 'island.is',
-    address: 'island@island.is',
+    sender: process.env.NODE_ENV === 'production' ? 'island.is' : 'devland.is',
+    address:
+      process.env.NODE_ENV === 'production'
```

Comment:
NODE_ENV will be production for dev, staging and prod

Reply:
applied this just now